templates/home.html

```diff
@@ -59,7 +59,7 @@ <h2 class="white-color title">Our Purpose</h2>
             <div class="discover extra-padding">
                 <h4 class="color-blue">Join the movement</h4>
                 <div class="center extra-padding">
-                    <a class="button sign-up" href="{% url 'register' %}?next={{ request.get_full_path }}">Sign Up</a>
+                    <a class="button sign-up" href="{% url 'register' %}?next={{ request.get_full_path }}">START INVESTING</a>
```

Comment:
Remember that the ALL CAPS causes Google and other search engines to index the text incorrectly, so we really should be using CSS to make the text uppercase instead.